### PR TITLE
CORTX-29719: Set proper setup size for services other than ios and confd

### PR DESCRIPTION
**Pull request by atulsdeshmukh2312**
…nfd in mote mini provisioner

Signed-off-by: Atul Deshmukh <atul.deshmukh@seagate.com>

# Problem Statement
Set proper setup size for services other than ios and confd

# Design
1: We set the setup size according to mem limits for each motr service. This mem limits are available in /etc/cortx/cluster.conf
2: Currently, these limits are available only for ios and confd motr services. 
3: If provisioner pass service parameter (to motr_setup command during deployment phases) is other than confd and ios, it tends to fail to set the setup_size. To overcome this, we ignore these (non ios and non confd) services and just set the setup_size to small to services other than ios and confd.

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
 We deployed the cluster successfully. 
 In deployment,  we use (num_client_inst: 1) in solution.yaml to create motr clients.
 To create client pod, provisioner would pass service as 'client' to motr setup phases. In our fix, once we see service is client ( which is except ios, confd) we ignore it and set the setup size to small to avoid failures.  

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/libexec/platform-python
```
Comment (madhavemuri) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:1]: Why this change is done ?

**Review by madhavemuri — APPROVED** [review]

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 2]
```diff
@@ -234,14 +234,21 @@ def calc_size(self, sz):
         self.logger.error("Please use valid format Ex: 1024, 1Ki, 1Mi, 1Gi etc..\n")
         return ret
 
-def get_setup_size(self, service):
+def set_setup_size(self, service):
```
Comment (yeshpal-jain-seagate) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:237]: can we set setup_size as small by default, if it is not any of this case than it would be always small.

Comment (atulsdeshmukh2312) [reply scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:237]: Resolved in 41f0939a627000ba5a567ba4b037bfb7f013ac2a

**Review by yeshpal-jain-seagate — APPROVED** [review]
